app/assets/stylesheets/application.css.scss

```diff
@@ -24,3 +24,12 @@
   padding-left: 50px;
   padding-right: 50px;
 }
+
+.navbar-default .navbar-nav > li > a  {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 4<br>Opening curly brace `{` should be preceded by one space